R/utils.R

```diff
@@ -160,3 +160,36 @@ region_dispatch <- function(level, all_levels, region_names, region_codes) {
   region_vars <- region_vars[!is.na(region_vars)]
   return(region_vars)
 }
+
+#' Constructs list of options for processing
+#'
+#' @description Contructs processing options including user arguments.
+#' @param process_options list, additional arguments to control what
+#' functions are called during processing. For some datasets setting these
+#' to FALSE may cause errors, but for others improve speed.
+format_process_options <- function(process_options) {
+  base_options <- list(
+    "calculate_columns_from_existing_data" = TRUE,
```

Comment:
not sure we should support options that will always break?

Reply:
Agreed